roles/neutron/vars/main.yml

```diff
@@ -60,7 +60,7 @@ __neutron_helm_values:
         service_provider: VPN:strongswan:neutron_vpnaas.services.vpn.service_drivers.ipsec.IPsecVPNDriver:default
     dhcp_agent:
       DEFAULT:
-        dnsmasq_dns_servers: "{{ neutron_coredns_cluster_ip | default('10.96.0.20') }}"
+        dnsmasq_dns_servers: "{{ neutron_coredns_cluster_ip | default('1.1.1.1') }}"
```

Comment:
When using ML2/OVS, the `10.96.0.20` works just fine, but when using ML2/OVN we need to use `1.1.1.1`.

Reply:
I reverted back for ML2/OVS to 10.96.0.20